src/registry_testing.jl

```diff
@@ -251,3 +251,23 @@ function test(path=pwd(); registry_deps::Vector{<:AbstractString}=String[])
     end
     return nothing
 end
+
+# Change all occurences of "digit-digit" to "digit - digit"
+function _spacify_hyphens(str::AbstractString)
```

Comment:
Could we instead just strip all white space from both sides? I think that would be less fragile in case there’s other white space changes later

Reply:
Or just remove all white space before comparison?